.jest/fastify.env.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+import NodeEnvironment from "jest-environment-node";
+import { FastifyInstance } from "fastify";
+import { buildAPI } from "../src/api";
+
+export default class FastifyEnvironment extends NodeEnvironment {
+
+  public async setup(): Promise<void> {
+    await super.setup();
+    const fastify = await buildAPI();
+    this.global.fastify = fastify;
+  }
+
+  public async teardown(): Promise<void> {
+    (<FastifyInstance>this.global.fastify).close();
+    await super.teardown();
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
Yes... we gonna add lint/formatting rules to not have this kind of issues

Reply:
Absolutely. I added suggestions so you can just confirm them for quick fixes.

Reply:
Once this PR is merged I'll add tslint to a separate one to make this easier.

Reply:
Sounds great